src/Domain/Insights/SyntaxCheck.php

```diff
@@ -83,6 +67,47 @@ public function process(): void
         }
     }
 
+    private function getBinary(): string
+    {
+        $parentPath = $this->vendorParentPathFind();
+
+        if (DIRECTORY_SEPARATOR === '\\') {
+            return $parentPath . '\vendor\bin\parallel-lint.bat';
+        }
+
+        return sprintf(
+            '%s %s',
+            escapeshellcmd((string) Config::getExecutablePath('php')),
+            escapeshellarg($parentPath . '/vendor/bin/parallel-lint')
+        );
+    }
+
+    /**
+     * @return array<string>
+     */
+    private function getShellExcludeArgs(): array
+    {
+        $configuration = Container::make()->get(Configuration::class);
+
+        $rootExcludes = $configuration->getExcludes();
```

Comment:
Normally these would be excluded in the files collector but, since #437, SyntaxCheck no longer uses that so we need to grab the top-level excludes manually for inclusion in the arg list.